src/api/controllers/clinical/assessment/assessment.controller.ts

```diff
@@ -277,6 +281,80 @@ export class AssessmentController extends BaseController{
         }
     };
 
+    answerQuestionList = async (request: express.Request, response: express.Response): Promise<void> => {
+        try {
+            
+            await this.setContext('Assessment.AnswerQuestion', request, response);
+
```

Comment:
Add check whether all answers are sent at the start only.

Reply:
done